framework/src/modules/chain/chain.js

```diff
@@ -511,7 +511,7 @@ module.exports = class Chain {
 		this.scope.modules.synchronizer = this.synchronizer;
 	}
 
-	_startLoader() {
+	async _startLoader() {
```

Comment:
if this is asyc, the line below shouldve been awaited

Reply:
returning the promise now, and waiting in the subscribe, so should be good.